runtime/testy/studentAPI.py

```diff
@@ -7,16 +7,17 @@ def __init__(self, toManager, fromManager):
     # Wait for ack from SM before continuing
     self.fromManager.recv()
 
-  def getValue(self, key):
+  def getValue(self, *args):
     """Returns the value associated with key
     """
-    # TODO: Actually use key
-    self.toManager.put([SM_COMMANDS.GET_VAL, key])
+    keys = args
+    self.toManager.put([SM_COMMANDS.GET_VAL, keys])
     return self.fromManager.recv()
 
-  def setValue(self, key, value):
+  def setValue(self, *args):
     """Sets the value associated with key
     """
-    # TODO: Implement
-    self.toManager.put([SM_COMMANDS.SET_VAL, key, value])
+    value = args[-1] if len(args) > 0 else None
```

Comment:
In theory everything making it to stateManager should be parsed already.
Since this is studentAPI, we should throw an exception. The exception is being worked on by Carlos, Justin, Legeng.

One could probably also rewrite the argument list to be (value, key1, [additional keys])